mleap-tensorflow/src/test/scala/ml/combust/mleap/tensorflow/converter/MleapConverterSpec.scala

```diff
@@ -0,0 +1,192 @@
+package ml.combust.mleap.tensorflow.converter
+
+import ml.combust.mleap.tensor.{ByteString, Tensor}
+import ml.combust.mleap.core.types.TensorType
+import org.scalatest.FunSpec
+import org.tensorflow.types.TFloat32
+import org.tensorflow.types.TFloat64
+import org.tensorflow.types.TInt32
+import org.tensorflow.types.TInt64
+import org.tensorflow.types.TString
+import org.tensorflow.types.TUint8
+
+object Helper {
+
+  def scalars(dimensions: Traversable[Int]): Seq[Seq[Int]] = {
+    combine(dimensions.map(d => 0 until d))
+  }
+
+  def combine[A](xs: Traversable[Traversable[A]]): Seq[Seq[A]] =
+    xs.foldLeft(Seq(Seq.empty[A])) {
+      (x, y) => for (a <- x.view; b <- y) yield a :+ b
+    }
+}
+
+class MleapConverterSpec extends FunSpec {
+
+  describe("round trip from mleap tensor to tensorflow tensor") {
+
+    it("vector") {
+      val array = Array(20, 10, 5)
+      val ml_tensor = Tensor.denseVector(array)
+      val tf_tensor_converted = MleapConverter.convert(ml_tensor, TensorType.Int())
+      val ml_tensor_converted = TensorflowConverter.convert(tf_tensor_converted, TensorType.Int())
+      assert(ml_tensor.dimensions == ml_tensor_converted.dimensions)
+      assert(ml_tensor.base == ml_tensor_converted.base)
+      assert(ml_tensor.size == ml_tensor.size)
+      assert(ml_tensor.toArray sameElements ml_tensor_converted.toArray)
```

Comment:
could consider putting these 4 lines as a helper method since they get repeated so often.  Or even adding an equals method onto mleap tensor so you can just `assert(ml_tensor == ml_tensor_converted)`